commitlint.config.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-    extends: ['cz', '@commitlint/config-lerna-scopes'],
+    extends: ['@commitlint/config-conventional'],
     rules: {
```

Comment:
Let's remove rules, we don't need it and use defaults, also we need fix our doc, `ast` is removed and `tests` renamed to `test`

Reply:
yep lets do it

Reply:
removing all rules

Reply:
Done